{
  "info": {
    "author": "",
    "author_email": "",
    "bugtrack_url": null,
    "classifiers": [],
    "description": "",
    "description_content_type": "",
    "docs_url": null,
    "download_url": "",
    "downloads": {
      "last_day": -1,
      "last_month": -1,
      "last_week": -1
    },
    "home_page": "",
    "keywords": "",
    "license": "",
    "maintainer": "",
    "maintainer_email": "",
    "name": "slack-exception-messages",
    "package_url": "https://pypi.org/project/slack-exception-messages/",
    "platform": null,
    "project_url": "https://pypi.org/project/slack-exception-messages/",
    "project_urls": null,
    "release_url": "https://pypi.org/project/slack-exception-messages/0.1/",
    "requires_dist": [
      "slack-sdk (>=3.13.0)"
    ],
    "requires_python": ">=3.6",
    "summary": "A package that enables you to easily exception messages from django to slack",
    "version": "0.1",
    "yanked": false,
    "yanked_reason": null
  },
  "last_serial": 16407819,
  "releases": {
    "0.1": [
      {
        "comment_text": "",
        "digests": {
          "blake2b_256": "0dcf5a5711d05277064e03b1d9b3f75e17b3d1033303b15f6eb866f5c682ab21",
          "md5": "48ff3ddd192c7ffd5fa4b15be41f4b09",
          "sha256": "127fdf4a6166e3de18795664f798304b1622b4b89b458e7550c79e6f71a6f2b2"
        },
        "downloads": -1,
        "filename": "slack_exception_messages-0.1-py3-none-any.whl",
        "has_sig": false,
        "md5_digest": "48ff3ddd192c7ffd5fa4b15be41f4b09",
        "packagetype": "bdist_wheel",
        "python_version": "py3",
        "requires_python": ">=3.6",
        "size": 3364,
        "upload_time": "2023-01-12T18:45:18",
        "upload_time_iso_8601": "2023-01-12T18:45:18.114069Z",
        "url": "https://files.pythonhosted.org/packages/0d/cf/5a5711d05277064e03b1d9b3f75e17b3d1033303b15f6eb866f5c682ab21/slack_exception_messages-0.1-py3-none-any.whl",
        "yanked": false,
        "yanked_reason": null
      }
    ]
  },
  "urls": [
    {
      "comment_text": "",
      "digests": {
        "blake2b_256": "0dcf5a5711d05277064e03b1d9b3f75e17b3d1033303b15f6eb866f5c682ab21",
        "md5": "48ff3ddd192c7ffd5fa4b15be41f4b09",
        "sha256": "127fdf4a6166e3de18795664f798304b1622b4b89b458e7550c79e6f71a6f2b2"
      },
      "downloads": -1,
      "filename": "slack_exception_messages-0.1-py3-none-any.whl",
      "has_sig": false,
      "md5_digest": "48ff3ddd192c7ffd5fa4b15be41f4b09",
      "packagetype": "bdist_wheel",
      "python_version": "py3",
      "requires_python": ">=3.6",
      "size": 3364,
      "upload_time": "2023-01-12T18:45:18",
      "upload_time_iso_8601": "2023-01-12T18:45:18.114069Z",
      "url": "https://files.pythonhosted.org/packages/0d/cf/5a5711d05277064e03b1d9b3f75e17b3d1033303b15f6eb866f5c682ab21/slack_exception_messages-0.1-py3-none-any.whl",
      "yanked": false,
      "yanked_reason": null
    }
  ],
  "vulnerabilities": []
}